python/cryptography/security/insecure-cipher-mode-ecb.yaml

```diff
@@ -1,6 +1,9 @@
 rules:
   - id: insecure-cipher-mode-ecb
     pattern: cryptography.hazmat.primitives.ciphers.modes.ECB(...)
+    fix-regex:
+      regex: ECB
```

Comment:
This will break AES functionality - ECB does not require an IV (all blocks are encrypted identically) and ECB mode will be missing correct IV initialization for CBC mode.